tests/ImageSharp.Tests/Colorspaces/RgbTests.cs

```diff
@@ -49,7 +49,7 @@ public void RgbAndRgb24Operators()
             const byte g = 128;
             const byte b = 255;
 
-            Rgb24 rgb24 = new Rgb(r / 255F, g / 255F, b / 255F);
+            var rgb24 = new Rgb(r / 255F, g / 255F, b / 255F);
```

Comment:
Bug is in the test itself. We're not implicitly casting anymore.

Reply:
will revert that one